libkbfs/block_journal.go

```diff
@@ -397,8 +397,18 @@ func (j *blockJournal) getAll() (
 
 func (j *blockJournal) putData(
 	id BlockID, context BlockContext, buf []byte,
-	serverHalf BlockCryptKeyServerHalf) error {
-	err := validateBlockServerPut(j.crypto, id, context, buf)
+	serverHalf BlockCryptKeyServerHalf) (err error) {
+	j.log.Debug("Putting %d bytes of data for block %s with context %v",
```

Comment:
I definitely think it's worth plumbing `ctx` through to all these log messages.  It makes debugging so much easier.